config/secrets.yml

```diff
@@ -13,10 +13,15 @@
 development:
   secret_key_base: ca25c3e58abb023159f0315c3d04e0a44acd1eab0b6805f21c15379a581ebc9513621bcd6f7e133f7c16df237799263250d2533419ce882f5c90f5eb6ec86ff7
 
+  recaptcha_invisible_site_key: '6LfxjCMUAAAAAOk9kDKwAiAXCBugl3vnwU0_7Qrj'
+  recaptcha_invisible_secret_key: '6LfxjCMUAAAAAG_6BdL5W7cvrnUxygGc-9a8tmO_'
+
 test:
   secret_key_base: 737ecfed3ca74d8bba84c38c416f7c265295a3e1f939beeef4f4c8cdb6a9dc7c675a40ac72d10d5e47ad1b83129b53777f0daf12fb2c956c8878f8796df0398e
 
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  recaptcha_invisible_site_key: '6LfxjCMUAAAAAOk9kDKwAiAXCBugl3vnwU0_7Qrj'
```

Comment:
don't commit keys for production into git repositories. There is a comment directly above this section warning you to do exactly not this.